integration/copy_test.go

```diff
@@ -1114,10 +1114,12 @@ func (s *CopySuite) TestCopyVerifyingMirroredSignatures(c *check.C) {
 	// Create a signature for mirroring-primary:primary-signed without pushing there. This should be easier than using standalone-sign.
 	signingDir := filepath.Join(topDir, "signing-temp")
 	assertSkopeoSucceeds(c, "", "copy", "--src-tls-verify=false", regPrefix+"primary:unsigned", "dir:"+signingDir)
-	c.Logf("%s", combinedOutputOfCommand(c, "ls", "-laR", signingDir))
+	// Unknown error in Travis: https://github.com/containers/skopeo/issues/1093
+	//	c.Logf("%s", combinedOutputOfCommand(c, "ls", "-laR", signingDir))
```

Comment:
These are “just” diagnostic logs, but I’m not too happy to just ignore a weird failure like this. Let’s see what https://github.com/containers/skopeo/pull/1096 does.